runsc/container/multi_container_test.go

```diff
@@ -51,11 +51,20 @@ func createSpecs(cmds ...[]string) ([]*specs.Spec, []string) {
 				specutils.ContainerdContainerTypeAnnotation: specutils.ContainerdContainerTypeContainer,
 				specutils.ContainerdSandboxIDAnnotation:     rootID,
 			}
+			if i != 1 {
```

Comment:
It may come as a surprise to callers that some containers are using PID namespace and some are not. I'd recommend keeping createSpecs as before and make the caller (i.e. the test) to modify the specs to make it clear to the reader what is being done. For example, TestMultiContainerSanity would take spec[2] and add PID namespace to it.